c++/src/kj/async.c++

```diff
@@ -278,6 +267,16 @@ public:
     node->onReady(this);
   }
 
+  KJ_MUST_USE_RESULT Own<Task> pop() {
```

Comment:
Why `KJ_MUST_USE_RESULT`? Technically the call site in `~TaskSet()` would work just fine even if it _didn't_ store the result to a local, wouldn't it?

Reply:
Yeah. I added this when I needed the result but I found a way to simplify. I'll remove it.